src/main/kotlin/com/anaplan/engineering/vdmgradleplugin/OvertureWrapper.kt

```diff
@@ -421,18 +405,13 @@ private enum class ExpectedTestResult(val description: String) {
 }
 
 // ideally this would be done through an annotation, but this is not feasible currently
-private fun getExpectedResult(testName: String): ExpectedTestResult {
-    val testNameLower = testName.toLowerCase()
-    return if (testNameLower.toLowerCase().contains("expectpreconditionfailure")) {
```

Comment:
`toLowerCase` has been deprecated in Kotlin 1.5. Even though Gradle 7.x appears to force the use of Kotlin 1.4.y, using case-insensitive comparisons allows for a bit of future-proofing.